README.md

```diff
@@ -165,12 +165,12 @@ target:
 This will be applied in any process that has (`service_name == "frontend"` OR `service_name == "auth"`) AND `datacenter == "AWS-US-EAST-1"`.
 
 ### Matching Values
-By adding a backslash `/` at the front and end of a value, context values will be applied if the string between the backslashes is a substring of the value.
+By adding a slash `/` at the front and end of a value, context values will be applied if the string between the slashes is a substring of the value.
```

Comment:
The singular is still a bit confusing. And the substring part isn't right--it'll be a regexp match!
How about this wording:
```
To provide a regular expression for matching, surround it with a leading and trailing slash `/`. For example:
```
And then I'd make sure that the example is more than just a substring. That's why I suggested
```
  service_name: /frontend-\d/
```
and then maybe you can give an example that matches because of a digit and also one that fails because it's alpha there.